### ISSUE TITLE
DIRAC Support

### ISSUE THREAD

**Issue by hoganman** [t2k-software/t2kdm#21]
When using the DIRAC package, it comes with its own pip and pip install t2kdm does not work out of the box.

```bash
[mhogan@hep software]$ git clone https://github.com/t2k-software/t2kdm
Cloning into 't2kdm'...
remote: Counting objects: 740, done.
remote: Compressing objects: 100% (145/145), done.
remote: Total 740 (delta 177), reused 183 (delta 108), pack-reused 487
Receiving objects: 100% (740/740), 151.05 KiB | 985.00 KiB/s, done.
Resolving deltas: 100% (518/518), done.
[mhogan@hep software]$ cd t2kdm
[mhogan@hep t2kdm]$ which pip
~/software/dirac_ui/Linux_x86_64_glibc-2.12/bin/pip
[mhogan@hep t2kdm]$ pip install t2kdm
Collecting t2kdm
  Could not fetch URL https://pypi.python.org/simple/t2kdm/: There was a problem confirming the ssl certificate: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:661) - skipping
  Could not find a version that satisfies the requirement t2kdm (from versions: )
No matching distribution found for t2kdm
```

I even try to force the correct URL with no success.

```bash
[mhogan@hep t2kdm]$ pip install --index-url https://pypi.org/project t2kdm
Collecting t2kdm
  Could not fetch URL https://pypi.org/project/t2kdm/: There was a problem confirming the ssl certificate: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:661) - skipping
  Could not find a version that satisfies the requirement t2kdm (from versions: )
No matching distribution found for t2kdm
```

EDIT: I should mention my DIRAC setup. Following the directions on https://www.gridpp.ac.uk/wiki/Quick_Guide_to_Dirac#Dirac_client_installation , I installed DIRAC like so

```bash
./dirac-install -r v6r20p5 -i 27 -g v14r1
``` 

EDIT2: Fixed a word

**Comment by ast0815** [comment]
That looks like the Dirac pip is messing up the SSL connection to the PyPI server. Can you install any other Python packages?

**Comment by hoganman** [comment]
I tried to installed numpy and got the same SSL error. Even a bootstrap install of pip did not get pasted the SSL error. I tried various Google searches on similar issues and none resolve this. 

I verified that it is not my particular machine since I installed from source python2.7 and pip with it with an identical test with numpy

The only thing I can think of now is to test whether this is an OS issue. The "official" supported OS is SL6 and SL7, whereas I am using CENTOS 6.
